src/utils/utils.ts

```diff
@@ -95,11 +95,15 @@ export const splitArrEvery2 = (arr: string[]) => {
   return newArr;
 };
 
-// for edit item page
-// const [states, reasons] = ["System.State", "System.Reason"].map((e) => {
-//   return workItemTypeFields.data?.value
-//     ?.find((f) => f.referenceName === e)
-//     ?.allowedValues.map((f) => ({
-//       name: f,
-//     }));
-// });
+export const parseJsonErrorMessage = (error: string) => {
+  try {
+    const parsedError = JSON.parse(error);
+
+    if (parsedError.status === 401)
+      return "The user is not authorized to access this resource.";
```

Comment:
Perhaps we need a more user-friendly message here? Maybe talk to David about this as I think we need to include the following information:

* Which resource can't be accessed?
* How can the user access the resource / or if it's impossible, state that

Reply:
I had to do this because azure returns an HTML page as a response, so I cant really go by what kind of thing we cant access

Reply:
@iTweeno even with the request `Accept` header set to `application/json`?

Reply:
I can try and add it to see